### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,18 +1,18 @@
 Accessing CDAP Data from BI Tools
 ==================================
 
-The Cask Data Application Platform (CDAP) Datasets provide an abstraction to store data. In this guide you will learn
+The Cask Data Application Platform (CDAP) Datasets provide an abstraction (Datasets) to store data. In this guide, you will learn
```
Comment (anwar6953) [inline README.rst:4]: I think we should remove either the first instance of 'Datasets' or the second from this line.

File: README.rst [hunk 2]
```diff
@@ -238,14 +239,14 @@ A Custom Dataset is implemented to be RecordScannable, for integration with Hive
   }
 
 
-Build & Run
------------
+Build and Run
```
Comment (anwar6953) [inline README.rst:242]: For whatever reason this change happened, should we then also change the Share & Discuss section at the bottom?

File: README.rst (hunk 1 above)
Comment (awholegunch) [inline README.rst:4]: The Cask Data Application Platform (CDAP) provides an abstraction—Datasets—to store data.

File: README.rst (hunk 2 above)
Comment (awholegunch) [inline README.rst:242]: We should make them all consistent, and I would say "and". We don't generally use "&"; instead, we spell things out. I changed it as the first instance I found was "Build and Run", and as I wanted to turn it into a hyperlink, the text and tag needed to match.

File: README.rst [hunk 3]
```diff
@@ -264,27 +265,29 @@ Next, we will send some sample purchase events into the stream for processing::
...
   * What are the three most popular products?
 
-If you add zip code to the purchase event, then you can ask region-based questions such as -
-  * Which are the popular products in any region?
+If you add a zip code to the purchase event, you can then ask region-based questions, such as:
+
+  * Which are the popular products in a given region?
   * Which regions have the greatest revenue?
 
 Share & Discuss!
```
Comment (anwar6953) [inline README.rst:293]: Share and Discuss

**Pull request closed without merging**